### ISSUE TITLE
hide tab bar?

### ISSUE THREAD

**Issue by cypa** [lanoxx/tilda#322]
I use tilda primarily to evade all that sidethings like borders, bars, menus etc. Cause I prefer minimalistic interfaces, espesially on my netbook.
But tab bar is a big thing on the top which I dislike very much and I want to hide/remove it!
How do I do this?

thanks for tilda, it is cool, btw

**Comment by cypa** [comment]
https://github.com/lanoxx/tilda/issues/81

**Comment by cypa** [comment]
I've found https://github.com/lanoxx/tilda/issues/81 and it works OK ... but then I thought it would be better to add separate widget checkbox to tide tab bar cause position: hidden is not obvious at all. I looked through these settings many times and never tumble to check this position dropdown.

**Comment by MatthewScholefield** [comment]
Is this resolved? There's a check box in the settings to hide the tab bar.

**Comment by cypa** [comment]
Yes, resolved. Bur thats no checkbox , its a dropdown list with misleading caption ;)2:37, 22 февраля 2018 г., "Matthew D. Scholefield" <notifications@github.com>:Is this resolved? There's a check box in the settings to hide the tab bar.

—You are receiving this because you modified the open/close state.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




Отправлено из мобильной Яндекс.Почты: http://m.ya.ru/ymail
